trends/src/main/kotlin/net/thechance/trends/api/dto/trend/TrendResponse.kt

```diff
@@ -29,7 +29,9 @@ fun Trend.toResponse(isLiked: Boolean = false): TrendResponse {
         likesCount = likesCount,
         viewsCount = viewsCount,
         isCurrentUserOwner = false,
-        isLiked = isLiked
+        isLiked = isLiked,
+        username = owner?.username ?: "Unknown",
```

Comment:
Should we allow for such a trend to be on users' feed? wouldn't it be better to omit it from the response given its data is missing?